src/common.c

```diff
@@ -516,3 +516,391 @@ static SEXP from_table_kobject(K x) {
   make_data_frame(result);
   return result;
 }
+
```

Comment:
Looks like this is based on old version of common.c(which uses lists for attributes). Could you drop this and use sexp2k.c from https://github.com/KxSystems/rkdb/pull/5/? I can merge only that file from PR#5 and your changes except for common.c?